admin/github.go

```diff
@@ -216,15 +221,88 @@ func (s *Service) processGithubPush(ctx context.Context, event *github.PushEvent
 
 func (s *Service) processGithubInstallationEvent(ctx context.Context, event *github.InstallationEvent) error {
 	switch event.GetAction() {
-	case "created", "unsuspend", "suspend", "new_permissions_accepted":
+	case "created", "unsuspend", "new_permissions_accepted":
 		// TODO: Should we do anything for unsuspend?
-	case "deleted":
-	}
+	case "suspend", "deleted":
+		// the github installation ID will change if user re-installs the app deleting the project for now
+		installation := event.GetInstallation()
+		if installation == nil {
+			return fmt.Errorf("nil installation")
+		}
 
+		if err := s.deleteProjectsForInstallation(ctx, installation.GetID()); err != nil {
+			s.logger.Error("failed to delete project for installation", zap.Int64("installation_id", installation.GetID()), zap.Error(err))
+			return err
+		}
```

Comment:
1. Wrap with start/stop info logs
2. Maybe prefix all the new logs in this file with "github webhook: ...`
3. Missing `observability.ZapCtx(ctx)` field in all the logs